guides/WRITING-DOCUMENTATION.md

```diff
@@ -54,156 +68,165 @@ The description should describe what pattern does or should be used for in plain
+| **`@uswds [URL]`**          | `@uswds https://designsystem.digital.gov/components/button/` | `[URL]` is the link to the corresponding US Web Design System component.                                                                |
 
 ### Modifiers
 
 If the item you are documenting has multiple states or styles depending on added classes or pseudo-classes, you should document them in the modifiers section. [More info on modifiers from the KSS documentation site](https://warpspire.com/kss/syntax/).
 
 ### Markup
 
-Markup examples can be written in plain HTML inline or in a separate `.html` file, relative to the CSS file:
+Markup examples shold be written in a separate plain `.html` file, relative to the SCSS file:
 
 ```
 Markup: Button.example.html
 ```
 
-#### Button.example.html file
-
-```HTML
-<button class="ds-c-button {{modifier}}">Button label</button>
-```
-
-### Style guide: Reference
+### Style guide
 
 The `reference` defines the documentation site's structure.
 
-The expected format is:
-`[Top-level slug].[Subpage slug].[Page section slug]` for example `Style guide: components.button.disabled`
+The expected format is: `[Top-level slug].[Subpage slug].[Page section slug]`.
+For example, `Style guide: components.button.disabled`.
 
 The docs site supports a maximum of 2 levels of nesting. Pages nested a 3rd level will be displayed in the body of their parent page.
 
 Pages generated from KSS comment blocks are ordered alphabetically, and page sections are displayed in the order in which they are in the SCSS file.
 
-### Documentation and guidance
+### Guidance
+
+To add a guidance section to the page the `style guide:` `[Page section slug]` should be `guidance`.
+For example: `Style guide: components.button.guidance`
 
-To add guidance content to the page, the page section's slug should be `guidance`.
-For example: `components.buttons.guidance`
+Guidance should be written using [Markdown](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet) if you include `---` right after your opening section comments, `/*`.
 
-Guidance can be written using [Markdown](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet) if you include `---` right after your opening section comments, `/*`. For example
+To link to another documentation page in your guidance content use a relative url.
+For example, `/components/button`
 
-To link to another documentation page in your guidance content use a relative url. **Example:** `/components/button`
+Documentation and guidance should be placed at the end of the SCSS file and conform to the following format and sections:
 
-Documentation and guidance should be placed at the end of the SCSS file and conform to the following format:
+<details>
+<summary><b>View guidance section example</b></summary>
```

Comment:
Where did these come from?

Reply:
The guidance section was not in a collapsable details view previously - see here - https://github.com/CMSgov/design-system/pull/739/files#diff-36dd66eb15347ca0028585e9135a481cL126

Reply:
Ah I see, I never saw that collapseable view before.